pkg/middleware/org_redirect_test.go

```diff
@@ -30,7 +30,7 @@ func TestOrgRedirectMiddleware(t *testing.T) {
 		}
 
 		sc.m.Get("/", sc.defaultHandler)
-		sc.fakeReq("GET", "/?orgId=3").exec()
+		sc.fakeReq("GET", "/?orgId=3&kiosk").exec()
```

Comment:
I would test all the following inputs:
- `/?orgId=3`
- `/?orgId=3&kiosk`
- `/?kiosk=&orgId=3`
- `/?kiosk=tv&orgId=3`